build.gradle.kts

```diff
@@ -70,6 +71,7 @@ subprojects {
         plugin(JibConfigPlugin::class.java)
         plugin(ClassPathPlugin::class.java)
         plugin("org.sonarqube")
+        plugin("org.jetbrains.kotlin.kapt")
```

Comment:
Do we want to apply this to all projects, or just the ones that need it?

Also for info, apparently kapt is in maintenance mode: https://kotlinlang.org/docs/kapt.html.  Hopefully MapStruct can switch over to KSP, but they've had an issue open for a while: https://github.com/mapstruct/mapstruct/issues/2522